Pod/Classes/ios/NYTPhotosOverlayView.m

```diff
@@ -49,6 +50,10 @@ - (void)layoutSubviews {
     }];
     
     [super layoutSubviews];
+
+    if ([self.captionView respondsToSelector:@selector(setPreferredMaxLayoutWidth:)]) {
```

Comment:
Rob pointed this out to me via chat and he mentions it at https://github.com/robrix/RXCollections/issues/3#issuecomment-13699744 .

More authoritatively: see the runtime source/comments: https://github.com/opensource-apple/objc4/blob/17d47b3989923007fa052f346b05a32fca389465/runtime/objc-runtime-new.mm#L5744

(that's an unofficial mirror of the source, but linking to a line on apple's opensource site is impossible :confused: )

whereas `respondsToSelector:` doesn't lock anything: https://github.com/opensource-apple/objc4/blob/17d47b3989923007fa052f346b05a32fca389465/runtime/objc-class.mm#L640